### PR TITLE
Enabling RSpec/VoidExpect

### PR DESCRIPTION
**Pull request by robaiken**
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecvoidexpect

### CODE REVIEW

File: npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb [hunk 1]
```diff
@@ -87,7 +87,7 @@ def project_dependency_file(file_name)
         expect(Dir.children(".")).to match_array(
           %w(package.json .npmrc .yarnrc)
         )
-        expect(File.empty?(".npmrc"))
+        expect(File.read(".npmrc")).not_to be_empty
```
Comment (jurre) [inline npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: This inverts the expectation doesn't it?

Comment (jurre) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: Very confused why this doesn't break any of the tests 😬

Comment (robaiken) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: Yea, @jurre , it shouldn't work, but I don't want to delay this epic for a broken test. I can either skip the test or leave it as it is, but I will raise an issue.

Comment (jurre) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: I'm just surprised that it was passing before. Is there a subtle difference between `File.empty?` and `File.read.empty?` maybe?

Comment (landongrindheim) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: I assume `.npmrc` has a zero size. `File.read` won't blow up if the file exists (which it does, as shown in the expectation a few lines up), but is empty.

```plaintext
 dependabot@codespaces-a9309d:/app$ touch .npmrc
dependabot@codespaces-a9309d:/app$ irb
irb(main):001> File.read(".npmrc").empty?
=> true
```

Comment (landongrindheim) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: > Is this expected behaviour?

There are enough references in this thread that I'm not 100% sure which behavior you're referring to 🙈 

My hope is that the (perceived) intent of the test will remain. The expectation could be updated to maintain the current behavior with something like 👇 
```suggestion
        expect(File.empty?(".npmrc")).to be_truthy
```

Comment (robaiken) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: @landongrindheim it looks like `File.empty?(".npmrc")` returns `false`. 

> There are enough references in this thread that I'm not 100% sure which behavior you're referring to 🙈
Lol, sorry, you are right. The behavior I am referring to is whether .npmrc should exist in this scenario. It seems like the original test was not set up correctly as it did not have an assertion, so it passes every time. If that's the case, then it's weird that RSpec doesn't log that the test has no assertions.

Comment (landongrindheim) [reply npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/dependency_files_builder_spec.rb:90]: @robaiken 👍 In that case, I think it makes sense to make the test honest and ensure that the assertions are inline with the intent of the tests. Good find!

**Review by GarryHurleyJr — APPROVED** [review]

**Pull request merged**